src/classes/RD2_DataMigrationMapper.cls

```diff
@@ -131,6 +131,8 @@ public class RD2_DataMigrationMapper {
      */
     private void setInstallmentFields(npe03__Recurring_Donation__c rd) {
 
+        validateAmountCanBeCalculated(rd);
```

Comment:
Is there a reason to validate the amount at this line? How about checking the value before current line 156, ie before setting RD amount?

Reply:
I'm fine to change this if it helps with readability, but it does mean that we will sometimes do unnecessary processing.

Reply:
Let's keep related code together in a cluster ;)